script.js

```diff
@@ -0,0 +1,270 @@
+let selectFile = document.querySelector("#uploadFile");
+let newImg = document.querySelector("#prodImg");
+let searchP = document.querySelector("#search");
+
+selectFile.addEventListener("change", getImagePreview);
+function getImagePreview() {
+  let imageObject = selectFile.files[0]; // first file from files the user have selected
+  if (imageObject) {
+    newImg.style.display = "block";
+    const reader = new FileReader();
+    reader.addEventListener("load", function (e) {
+      let bookImgObj = e.target.result;
+      newImg.setAttribute("src", bookImgObj);
+    });
+    reader.readAsDataURL(imageObject); // read the binary files like images
+  } else {
+    newImg.style.display = "null";
+    newImg.setAttribute("src", "");
+  }
+}
+
+// validation function
+function validation() {
+  const pname = document.getElementById("name").value;
```

Comment:
Variables name should be more clear. ex. You should have named `pname` as `productName`.